lib/membrane/rtp/parser.ex

```diff
@@ -72,7 +72,13 @@ defmodule Membrane.RTP.Parser do
           %{packet: packet, has_padding?: has_padding?, total_header_size: total_header_size}} <-
            RTP.Packet.parse(payload, not state.secure?) do
```

Comment:
shouldn't it be
```suggestion
           RTP.Packet.parse(payload, state.secure?) do
```
?

Reply:
Oh, yup. I forgot about it as previously the parser option had `parse_payload?` name but I did not change the logic there...

Reply:
You can also move the logic of handling the RTP packet entirely to either `process_packet` or `handle_process`

Reply:
I don't think that this logic can be moved out of parsing as it take the `encrypted?` information to either strip padding or not. We can move the padding stripping out of packet parsing but I don't know we we want that.

Reply:
I basically mean to remove that function: https://github.com/membraneframework/membrane_rtp_plugin/blob/decryptor-packet-filtering/lib/membrane/rtp/parser.ex#L125

Reply:
done